### PR TITLE
Grant cards with updates

### PR DESCRIPTION
**Pull request by 1emu**
- Add grants updates to cards

image

- Improve grant dashboard components layout for mobile.

image

- Refactor ProposalUpdate component. Tested it still works as expected:

image

image

- Add mobile navigation

image

image

- Fix mobile filters display
 
image

image

### CODE REVIEW

File: src/pages/grants.css [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
   margin-top: 15px;
-  row-gap: 16px;
-  column-gap: 32px;
+  display: grid !important;
+  grid-template-columns: 48.33% 48.33%;
+  grid-row-gap: 16px;
+  grid-column-gap: 31px;
+}
+
+@media only screen and (min-width: 320px) and (max-width: 990px) {
```
Comment (andyesp) [inline src/pages/grants.css:11]: Is it possible to swap these properties from the .GrantsCards__Container classes so this media query is only `min-width: 991px` ?

File: src/entities/Proposal/routes.ts [hunk 2]
```diff
@@ -759,10 +782,10 @@ async function getGrants() {
 
         const contract = {
           symbol,
-          balance: parseInt(balance, 10) / 10 ** decimals,
-          vestedAmount: parseInt(vestedAmount, 10) / 10 ** decimals,
-          releasableAmount: parseInt(releasableAmount, 10) / 10 ** decimals,
-          released: parseInt(released, 10) / 10 ** decimals,
+          balance: Math.round(parseInt(balance, 10) / 10 ** decimals),
```
Comment (ncomerci) [inline src/entities/Proposal/routes.ts:785]: I would not specify the base in the `parseInt` function, by default it is set to 10 and the code looks cleaner. Just my point of view, what do you think?

**Review by andyesp — APPROVED** [review]
I'd love to see the EmptyProposalUpdate inside the ProposalUpdate as it is visually the same component with a different state.

File: src/components/Grants/Banner.css [hunk 3]
```diff
@@ -10,14 +10,28 @@
   gap: 50px;
 }
 
+@media only screen and (min-width: 320px) and (max-width: 990px) {
```
Comment (ncomerci) [inline src/components/Grants/Banner.css:13]: "Mobile first" approach!

File: src/components/Layout/BurgerMenu/BurgerMenuPushableLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+function BurgerMenuPushableLayout({ children }: Props) {
+  const burgerMenu = useBurgerMenu()
+  const responsive = useResponsive()
+  const isMobile = responsive({ maxWidth: Responsive.onlyMobile.maxWidth })
+
+  if (isMobile) {
+    return (
+      <div
+        className="Animated"
+        style={burgerMenu?.status.open ? { transform: `translateY(${burgerMenu.status.translate})` } : {}}
```
Comment (andyesp) [inline src/components/Layout/BurgerMenu/BurgerMenuPushableLayout.tsx:21]: I'd love to see this in a class BurgerMenu--open 😏

File: src/components/Layout/BurgerMenu/MobileNavigation.tsx [hunk 5]
```diff
@@ -1,65 +1,38 @@
...
+import locations from '../../../modules/locations'
+import { NavigationProps, NavigationTab } from '../Navigation'
 
 import './MobileNavigation.css'
-import { NavigationTab } from './Navigation'
 
-type SelectedButtonType = {
-  inverted: boolean
-  primary: boolean
+function getProps(tab: NavigationTab, activeTab?: NavigationTab) {
```
Comment (andyesp) [inline src/components/Layout/BurgerMenu/MobileNavigation.tsx:15]: Maybe `getButtonProps`?

File: src/components/Layout/BurgerMenu/BurgerMenuPushableLayout.tsx (hunk 4 above)
Comment (1emu) [reply src/components/Layout/BurgerMenu/BurgerMenuPushableLayout.tsx:21]: We can't because translate is a variable that can take many values :)

**Review by andyesp — APPROVED** [review]

**Pull request merged**